x-pack/test/api_integration/apis/monitoring/kibana/rules_and_actions/instance.js

```diff
@@ -36,6 +37,7 @@ export default function ({ getService }) {
         .set('kbn-xsrf', 'xxx')
         .send({ timeRange })
         .expect(200);
+      body.metrics = normalizeDataTypeDifferences(body.metrics, fixture);
```

Comment:
@XavierM I'm thinking those values you were getting were actually correct and we need to update the fixtures with those values instead of normalizing.  Since we aren't using a derivative unit we wouldn't expect the normalized response?

Reply:
Makes sense, I'm in there fixing the tests after I added the technical preview badge, I'll take care of this also.